packages/core/src/V2/nodes/console.ts

```diff
@@ -0,0 +1,55 @@
+import { INodePorts, IWatcherNode, IWatcherNodeConfig, IWatcherResult, WatcherEvents } from '../Node';
+import { tap } from 'rxjs/operators';
+import { firstValueFrom, ReplaySubject, Subscription, SubscriptionLike, Observer, Observable } from 'rxjs';
+
+export class WatcherResult implements IWatcherResult, Observer<any> {
```

Comment:
This, WatcherNode etc can be upgraded to its own file/colocate with a "WatcherNode" file?